README.md

```diff
@@ -36,84 +43,87 @@ Using Async & Await, to run a select statement & displaying the result set:
-  try {
-    let statement =  new Connection().connect().getStatement();
+  const connection = new Connection({ url: '*LOCAL' });
+  const statement = new Statement(connection);
 
-    let result = await statement.exec('SELECT * FROM MYSCHEMA.TABLE');
-    
-    console.log(`Select results: \n${JSON.stringify(result)}`);
+  const results = await statement.exec('SELECT * FROM QIWS.QCUSTCDT');
 
-  } catch(error) {
-       console.error(`Error was: \n${error.stack}`);
-    }
+  console.log(`results:\n ${JSON.stringify(results)}`);
 }
 
-execExample();
+execExample().catch((error) => {
+  console.error(error);
+});
 
 ```
 ### prepare bind execute
 Using Async & Await, to prepare, bind, and execute an insert statement:
 
 ```javascript
-const idbp = require('idb-pconnector');
-const {Connection} = idbp;
+const {
+  Connection, Statement, IN, NUMERIC, CHAR,
+} = require('idb-pconnector');
 
 async function pbeExample() {
-  try {
-    let statement =  new Connection().connect().getStatement();
+  const connection = new Connection({ url: '*LOCAL' });
 
-    await statement.prepare('INSERT INTO MYSCHEMA.TABLE VALUES (?,?)');
+  const statement = new Statement(connection);
 
-    await statement.bindParam([
-      [2018, idbp.IN, idbp.INT],
-      ['example', idbp.IN, idbp.CHAR]
-    ]);
-    await statement.execute();
+  const sql = 'INSERT INTO QIWS.QCUSTCDT VALUES (?,?,?,?,?,?,?,?,?,?,?) with NONE';
 
-  } catch(error) {
-       console.error(`Error was: \n${error.stack}`);
-    }
+  await statement.prepare(sql);
+
+  await statement.bindParam([
+    [9997, IN, NUMERIC],
+    ['Johnson', IN, CHAR],
+    ['A J', IN, CHAR],
+    ['453 Example', IN, CHAR],
+    ['Fort', IN, CHAR],
+    ['TN', IN, CHAR],
+    [37211, IN, NUMERIC],
+    [1000, IN, NUMERIC],
+    [1, IN, NUMERIC],
+    [150, IN, NUMERIC],
+    [0.00, IN, NUMERIC],
+  ]);
+
+  await statement.execute();
 }
 
-pbeExample();
+pbeExample().catch((error) => {
+  console.error(error);
+});
 
 ```
 ### DBPool
 
 Using DBPool to attach a connection , execute a stored procedure , and finally detach the connection.
 
 ```javascript
-const {DBPool} = require('idb-pconnector');
-const pool = new DBPool();
-
-async function poolExample(){
- //attach() returns an available connection from the pool.
- let connection = pool.attach(),
-   statement = connection.getStatement(),
-   results = null;
-
- try {
-   await statement.prepare("CALL QIWS.GET_MEMBERS('QIWS','QCUSTCDT')");
-   await statement.execute();
-   results = await statement.fetchAll();
-
-   if (results !== null){
-     console.log(`\n\nResults: \n${JSON.stringify(results)}`);
-   }
-   //closes statements makes the Connection available for reuse.
-   await pool.detach(connection);
-
- } catch (error){
-   console.log(`Error was: \n\n${error.stack}`);
-   pool.retire(connection);
- }
-}
+const { DBPool } = require('idb-pconnector');
+
+async function poolExample() {
+  const pool = new DBPool({ url: '*LOCAL' });
```

Comment:
I understand this is not part of this change, but why does `DBPool` take a "url" parameter?

Reply:
[link to doc](https://github.com/IBM/nodejs-idb-pconnector/tree/master/docs#constructor-dbpooldatabase--config)

It used to determine the database to connect to.

Looking back 'url' might be the best word to use for that.

Reply:
url in this case defaults to *LOCAL therefore it can be left out in the example.

`const pool = new DBPool();`

Reply:
updated with requested changes